src/main/scala/io/sdkman/changelogs/ScalaCliMigrations.scala

```diff
@@ -0,0 +1,44 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "069")
+class ScalaCliMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001_add_scala-cli_candidate",
+    author = "lwronski"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "scala-cli",
+      name = "ScalaCLI",
```

Comment:
Could `"Scala CLI"` work? Don't know if they accept spaces here.
```suggestion
      name = "Scala CLI",
```